frontend/viewer/src/project/browse/EntriesList.svelte

```diff
@@ -62,6 +77,15 @@
     if (!entry) return;
     onSelectEntry(entry);
   }
+
+  async function handleDeleteEntry(entry: IEntry) {
+    const headword = writingSystemService.headword(entry);
+    const entryId = entry.id;
+    if (!await dialogsService.promptDelete($t`Entry: ${headword}`)) return;
```

Comment:
I find this:
![image](https://github.com/user-attachments/assets/5ace937a-7b75-40a9-8ebb-3faa25004c5e)

Tidier than this:
![image](https://github.com/user-attachments/assets/2e2bad85-6b53-4bd5-93fe-0b908148e050)

Though it's mostly the confirmation button that feels noisy to me. I'm neutral regarding whether the headword is in the title or not.

What do you think?

I went through the trouble of making the change, so I just pushed it, but we can revert it.

Reply:
Yeah I was annoyed by that too, decided to defer it but I'm fine to keep it since you did it already.